org.rajnisht7.PopSearch.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "org.rajnisht7.PopSearch",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "command": "/app/search_bar.sh",
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
it is either `x11` alone or `wayland` + `fallback-x11`.

Also `--share=ipc` is required for `x11` or `fallback-x11`